src/quo/components/overlay/view.cljs

```diff
@@ -0,0 +1,26 @@
+(ns quo.components.overlay.view
+  (:require
+    [quo.components.overlay.style :as style]
+    [quo.foundations.colors :as colors]
+    [react-native.blur :as blur]
+    [react-native.core :as rn]))
+
+(defn view
+  [{:keys [type]} child]
+  (let [background-color (case type
+                           :shell  colors/neutral-80-opa-80-blur
+                           :drawer colors/neutral-100-opa-70-blur)]
+    [rn/view
+     {:style (style/overlay-background background-color)}
+     (if (= type :shell)
+       [blur/view
+        {:blur-amount   20
+         :blur-radius   25
+         :blur-type     :transparent
+         :overlay-color :transparent
+         :style         style/container-style}
+        [rn/view {:style style/blur-style}
+         child]]
+       [rn/view
+        {:style style/container-style}
```

Comment:
For consistency and conciseness, the props can be on the same line as `rn/view` when they are not multiline.